src/Infrastructure.VS.UnitTests/SolutionInfoProviderTests.cs

```diff
@@ -64,19 +77,64 @@ public async Task Get_ReturnsExpectedValue(string solutionNameToReturn)
         }
 
         [TestMethod]
-        public async Task Get_ServiceCalledOnBackgroundThread()
```

Comment:
Miscellaneous: I renamed this existing test to `GetAsync_ServiceCalledOnUIThread`... line 94 in the changed version.